platform-sdk/swirlds-common/src/main/java/com/swirlds/common/threading/framework/internal/QueueThreadMetrics.java

```diff
@@ -56,7 +56,7 @@ public QueueThreadMetrics(@NonNull final AbstractQueueThreadConfiguration<?, ?>
      * @return the name of the busy time metric
      */
     public static String buildBusyTimeMetricName(@NonNull final String threadName) {
-        return "thread-busy-" + threadName;
+        return "thread_busy_" + threadName;
```

Comment:
Will this change metrics names that are current used?

Reply:
It will not change the name on the Prometheus side; they were already called this after the adapter fixed the name.